message.go

```diff
@@ -61,6 +63,22 @@ func (this *CommandMsg) FromSocket(sock *Socket) {
 
 		sock.Page = page
 		sock.Server.Store.SetPage(sock) // set new page
+	case "setgroups":
+		grps, ok := this.Command["groups"]
+		grps = strings.TrimSpace(strings.Replace(grps, " ", "", -1))
+		groups := strings.Split(grps, ",")
+
+		if !ok || len(groups) == 0 {
+			return
+		}
+
+		// hack check for empty slice
+		if len(sock.Groups) != 0 {
```

Comment:
Should this be moved above the `len(groups) == 0` so that an empty group command could remove all the socket from all groups?